portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,32 @@
+
+//this function generates a random quote to display, also making sure that the previously displayed random image is not the same as the current random image.
 function randomizeQuote() {
 
-  var imageIndex = Math.floor(Math.random() * 4) + 1;
+  //ImageNumber has the Image index of the quotes. This generates a number between [1-4]
+  var imageNumber = Math.floor(Math.random() * 4) + 1;
```

Comment:
Since 4 is used in different places, you want to make it a constant. It is also easier to maintain because you don't need to change every single 4 if you are adding more images later.

Reply:
Awesome. Will do that right away!